### PR TITLE
feat: options to require agreements to ts and cs and pp explicitly

### PR DESCRIPTION
**Pull request by royscheeren**
# Scope

Add options to the create message function to require users to explicitly agree to terms and conditions and/or privacy policy.

# Work done
Added an options object to the parameters of the createMessagePayload function to add the required sentences to the message.

### CODE REVIEW

**Comment by jdsika** [comment]
new linter options used? Can we put the linter options into the CONTRIBUTING.md?

File: src/siwt.ts [hunk 1]
```diff
@@ -98,11 +95,16 @@ export const getContractStorage = (network: Network) => (contractAddress: string
 
 export const _queryAccessControl =
   (contractStorage: (network: Network) => (x: string) => Promise<ContractLedgerItem[]>) =>
-  async ({ contractAddress, network = Network.ithacanet, parameters: { pkh }, test: { comparator, value } }: AccessControlQuery) => {
```
Comment (jdsika) [inline src/siwt.ts]: ghostnet

**Review by jdsika — CHANGES_REQUESTED** [review]

File: src/siwt.ts (hunk 1 above)
Comment (royscheeren) [reply src/siwt.ts]: This will change with #19

**Comment by jdsika** [comment]
@royscheeren I cannot see any changes in the signing message:

grafik

**Comment by jdsika** [comment]
I cannot see the policies in the message anymore. Please adjust the commands in Contribution.md on how to build this. I exchanged the line in index.js but I am confused by this message here:

> $ npm run demo:server:start
> npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
> 
> > @stakenow/siwt@0.0.5 demo:server:start
> > dotenv -e .env node ./demo/server/index.js
> 
> SIWT server app listening on port 3000

**Review by jdsika — APPROVED** [review]

**Pull request merged**